recipes/pytentiostat/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pytentiostat" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/juami/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 952e6e2743d88a3c519a9644f5514902f4b1726f67221727c71b7a37dd59ea5f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3
+    - setuptools
```

Comment:
These are not build deps

Reply:
Thanks @scopatz .  I copied this from the xonsh meta.yml.  Should there be no build requirements at all in that case (it is pure python, no c)?  In that case, do we put an empty list or just delete the ``build:``

Reply:
Yep, there are no build deps.  xonsh's is probably outdated